### PR TITLE
implement add assignment button + add assignment component

### PR DESCRIPTION
**Pull request by ryan-barrett**
https://csumb.instructure.com/courses/13619/assignments/101626?module_item_id=530135

### CODE REVIEW

File: src/components/AddAssignment.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    event.preventDefault();
+    switch (event.target.id) {
+      case 'name':
+        this.setState({ name: event.target.value });
+        break;
```
Comment (ghost) [inline src/components/AddAssignment.js:21]: Maybe instead of break, you can put like " "

File: src/components/AddAssignment.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    const { redirect } = this.state;
+
+    if (redirect) {
+      return <Redirect to='/'/>;
+    }
+
+    return (
+      <div align="left">
+        <h4 style={{ marginLeft: '3rem' }}>Add Assignment</h4>
+        <form onSubmit={this.handleSubmit} style={{
```
Comment (ghost) [inline src/components/AddAssignment.js:71]: Maybe for this you can also do a return due date and course?

File: src/components/AddAssignment.js (hunk 1 above)
Comment (ryan-barrett) [reply src/components/AddAssignment.js:21]: I think break is required for a switch case:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

File: src/components/AddAssignment.js (hunk 2 above)
Comment (ryan-barrett) [reply src/components/AddAssignment.js:71]: I don't understand this one. What do you mean? 🤔

**Pull request merged**